### PR TITLE
New Registration Screen

### PR DESCRIPTION
**Pull request by edencakir**
- Created [Registration Screen](https://projects.invisionapp.com/share/GBBRMN5UT#/screens/249811219)
- Style & folder structure work on top of #48 
- New component **Button** introduced. We can use this app wide.
- Laggy ticker view implemented ([looking for](https://stackoverflow.com/questions/46214814/scrolling-continuously-without-user-interaction) a better implementation)

About improving scroll animation performance; if these get merged I guess these animations will be way better. Until then we can also consider using gifs or a video:
https://github.com/facebook/react-native/pull/15609
https://github.com/facebook/react-native/pull/15610

Android **inverted flatlist issue**: https://github.com/facebook/react-native/pull/14560

simulator screen shot 14 sep 2017 5 11 08 pm

### CODE REVIEW

**Comment by edencakir** [comment]
@BilalBudhani @joshmadewell it'd be great if you guys can review this.

File: src/screens/Onboarding/OnboardingScreen.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import React from 'react';
+<<<<<<< HEAD
+import { View, ScrollView, Dimensions } from 'react-native';
+import { intro1, intro2, intro3, intro4 } from 'kitsu/assets/img/onboarding/';
+import { Button } from 'kitsu/components/Button';
+=======
 import { View, Text, TouchableOpacity, ScrollView, Dimensions } from 'react-native';
 import { intro1, intro2, intro3, intro4 } from 'kitsu/assets/img/onboarding/';
+>>>>>>> ac7ac011c40b7d0948eef59ca59ff6b91fec105f
```
Comment (BilalBudhani) [inline src/screens/Onboarding/OnboardingScreen.js]: @edencakir you missed resolving a conflict here.

Comment (edencakir) [reply src/screens/Onboarding/OnboardingScreen.js]: Ah thanks for the heads up. I'm not sure how I missed that.

**Comment by BilalBudhani** [comment]
@edencakir when I try to sign in with my username & password; nothing happens after pressing sign in button, I just see loading indicator on the button forever. Please look into this.

**Comment by edencakir** [comment]
@BilalBudhani This PR has nothing to do with Login or Signup screens (except facebook login). This is just a landing screen after onboarding. So for that, we should better look at the login screen.

**Comment by BilalBudhani** [comment]
@edencakir Okay. I think #66 branch by @joshmadewell fixed that issue.

**Comment by edencakir** [comment]
@BilalBudhani Great, let me go through this PR before merging.

**Comment by edencakir** [comment]
@BilalBudhani Done, should be ready to merge

**Pull request merged**